controllers/sriov-controller/README.md

```diff
@@ -0,0 +1,101 @@
+# SRIOV controller
```

Comment:
Can we add something at the top indicating this is untested code, and that we are in the process of adding CI to it? I want to make it clear it's not only experimental but also untested.